docs/service-catalog/docs/03-04-experimental-features.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Experimental features
+type: Details
+---
+
+Currently Service Catalog requires its own instance of api-server and etcd. This adds additional complexity to the cluster configuration and increases
```

Comment:
besides complexity and configuration please also mention that in case of APi-server downtime, all of the SC resources are not available (and from time to time it happens, simple pod restart can result with such behaviour)